benchmarks/k8s/base/agent-configurations/idle-no-monitors/agent-config-additional-fields

```diff
@@ -0,0 +1 @@
+"implicit_metric_monitor": false, "implicit_agent_process_metrics_monitor": false,
```

Comment:
This works for now, although I would prefer us to use config arguments and agent.d directory (aka the same we do for CodeSpeed benchmarks).

Any particular reason why we didn't do this here? Perhaps because we would need to share more files with the Docker container or similar?